assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -145,19 +145,19 @@ function AssignmentPlanningV2 (props) {
                             label: 'Current',
                             value: currentGrade,
                             color: 'steelblue',
-                            labelPlacement: 'down'
+                            labelPlacement: 'down1'
                           },
                           {
                             label: 'Goal',
                             value: goalGrade,
                             color: 'green',
-                            labelPlacement: 'up'
+                            labelPlacement: 'up1'
```

Comment:
text like `goal_up`??

Reply:
If you named it 'goal_up' and wanted to change where the labels went for each line you'd have to change the strings again to make sense vs just swapping 'down1' and 'down2'

I think it makes sense the way it is?